framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1841,25 +1868,37 @@ public void postAsMemberOf(
      * Adds default qualifiers bases on the underlying type of {@code type} to {@code type}. If
      * {@code element} is a local variable, then the defaults are not added.
      *
+     * <p>(This uses both the {@link DefaultQualifierForUseTypeAnnotator} and {@link
+     * DefaultForTypeAnnotator}.)
+     *
      * @param element possibly null element whose type is {@code type}
      * @param type the type to which defaults are added
      */
+    // TODO: rename in the next minor release.
```

Comment:
The August release will be the next minor release. So isn't now the right time to make this change?

Reply:
Yes, I didn't realize breaking changes were merged.  I'll fix it.